kibbeh/src/icons/LoadingIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+function LoadingIcon(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg
+      className="animate-spin h-4 w-4 text-primary-100"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+    >
+      <circle
+        className="opacity-25"
+        cx="12"
+        cy="12"
+        r="10"
+        stroke="currentColor"
+        stroke-width="4"
```

Comment:
should be strokeWidth. that will fix the failing checks.

Reply:
Done thanks.